### ISSUE TITLE
Allow the atmospheric transmission curves to vary by airmass

### ISSUE THREAD

**Issue by cwwalter** [LSSTDESC/imSim#179]
imSim uses transmission curves from the project controlled throughputs repo. 

The atmospheric curves are located here.

https://github.com/lsst/throughputs/tree/master/atmos

They are produced with the MODTRAN model set to typical expected conditions at Cerro Pachon including added aerosols. 

Currently the bandpasses loaded by the sims code are fixed to be at airmass=1.2.  However the throughputs directory does include files for air-masses between 1.0 and 2.5.  @danielsf points out that  
"There are methods in `sims_photUtils` to take a hardware-only bandpass (which also exists in `throughputs`) and multiply it by an arbitrary airmass transmission curve."

Creating code to leverage those routines would increase the realism of the atmospheric model and would be useful for the PC group etc.

**Comment by cwwalter** [comment]
@jmeyers314 Note this existing issue related to airmass varying throughput.  

Should we replace this with your suggestion of a multiplicative function and is thus something we can implement quickly for the release candidate?

**Comment by jmeyers314** [comment]
Yeah, I suspect we can take the total throughput files from throughputs/baseline, divide out the atmos_12.dat file and multiply in an interpolation of the nearest atmos_XY.dat files from throughputs/atmos.

**Comment by cwwalter** [comment]
This is a sprint day project on Friday.
